config/sources/families/rockchip-rk3588.conf

```diff
@@ -33,8 +33,8 @@ case $BRANCH in
 		SKIP_BOOTSPLASH="yes"
 		LINUXFAMILY=rockchip-rk3588
 		LINUXCONFIG='linux-rockchip-rk3588-'$BRANCH
-		KERNEL_MAJOR_MINOR="6.5" # Major and minor versions of this kernel.
-		KERNELBRANCH='tag:v6.5-rc5'
+		KERNEL_MAJOR_MINOR="6.6" # Major and minor versions of this kernel.
+		KERNELBRANCH='tag:v6.6-rc1'
```

Comment:
In a perfect world we'd commit this family change on a seperate PR and let the remaining changes be contingent on merge.

Maybe next time

Reply:
Yes. I hope this also resolves rtw88 issues that are related to this.